src/components/about/member.astro

```diff
@@ -7,7 +7,7 @@ const { name, link, profile_image } = Astro.props;
 <a href={link} class="block" target="_blank">
   <div class="md:mb mb-2">
     <Image
-      class="mx-auto w-full overflow-hidden rounded-t-[4px]"
+      class="aspect-[297/330] w-full overflow-hidden rounded-t-[4px]"
```

Comment:
Check the preview; it looks like some images are a bit stretched. @achrafhardizi  you may need to add `object-cover` to class to prevent that

Reply:
Fixed @yjose , I chose to amend the changes instead of creating a new commit because the update was minor